simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/configuration/SimulatorConfiguration.java

```diff
@@ -42,6 +46,25 @@
         private int connectors;
     }
 
+    @Data
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class MeterValuesConfiguration {
+        /**
+         * How often send meter values in seconds.
+         * If set to 0 then meter values are never sent.
+         */
+        @Builder.Default
+        private long sendMeterValuesIntervalSec = DEFAULT_SEND_METER_VALUES_INTERVAL_SEC;
+
+        /**
+         * Power consumed for each power consumption interval
```

Comment:
It's not per send interval anymore. Since we're using Wh - it's always an hour, so I'd change comment simply to 'Power consumption'